src/design-system/typography/fontWeights.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+export const fontWeights = {
```

Comment:
Did you test that all of them matter on both platforms? I was observing some weirdnesses with them.

Reply:
Yep, I tested these across both platforms. Just realised there's no example of the font weight scale in `Text.docs.ts` so I'll add that.

Reply:
Added examples to the repo. Here's some screenshots.

iOS:
<img width="201" alt="Screen Shot 2021-11-24 at 8 46 19 am" src="https://user-images.githubusercontent.com/696693/143134544-949df885-d8f2-4004-927c-c0b444551427.png">

Android:
<img width="192" alt="Screen Shot 2021-11-24 at 8 46 29 am" src="https://user-images.githubusercontent.com/696693/143134534-2a4fd008-d562-46b3-9dc2-e955e898012c.png">

Reply:
can't believe, it's so perfect